source/agora/cli/multi/main.d

```diff
@@ -138,7 +148,10 @@ private int main (string[] args)
 
 private void printDefaultHelp ()
 {
-    writeln("usage: agora-multi [path] node0.yaml node1.yaml node2.yaml ...");
+    writeln("usage: agora-multi command [path] node0.yaml node1.yaml node2.yaml ...");
+    writeln("These are commands:");
+    writeln("   start : Starts the nodes");
+    writeln("   test : Test connections to the nodes");
```

Comment:
I think you can do it generally with `Log.info`

Reply:
Thank you very much for your opinion.
There is no variety of outputs.
The output is all about help for the input and errors during execution.
When more different types of outputs occur, it would be better to use `Log`.